jsr352/core/src/test/java/org/hibernate/search/jsr352/massindexing/JobParameterValidationListenerTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.jsr352.massindexing;
+
+import org.hibernate.search.exception.SearchException;
+
+import org.junit.Test;
```

Comment:
I'm not sure if I should have an blank line between the two import statements above. I always have doubt about my Hibernate code style configuration in IntelliJ...